### ISSUE TITLE
QA instance-rename fails for non-LVM disk-templates

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#469]
Originally reported of Google Code with ID 413.

```
<b>What steps will reproduce the problem?</b>
1. Run QA with file disk-templates enabled

<b>What is the expected output? What do you see instead?</b>

---- 2013-04-05 17:11:50.970781 time=0:00:14.411483 gnt-instance rename
Traceback (most recent call last):
  File "./ganeti-qa.py", line 781, in <module>
    main()
  File "/buildbot/qa-kvm/qa-kvm-quick/build/ganeti/rapi/client.py", line 245, in wrapper
    return fn(*args, **kwargs)
  File "./ganeti-qa.py", line 776, in main
    RunQa()
  File "./ganeti-qa.py", line 690, in RunQa
    RunInstanceTests()
  File "./ganeti-qa.py", line 604, in RunInstanceTests
    RunCommonInstanceTests(instance)
  File "./ganeti-qa.py", line 297, in RunCommonInstanceTests
    rename_source, rename_target)
  File "./ganeti-qa.py", line 91, in RunTest
    retval = fn(*args, **kwargs)
  File "/buildbot/qa-kvm/qa-kvm-quick/build/qa/qa_utils.py", line 707, in wrapper
    result = fn(*args, **kwargs)
  File "/buildbot/qa-kvm/qa-kvm-quick/build/qa/qa_instance.py", line 423, in TestInstanceRenameAndBack
    AssertCommand(tags_cmd + rename_target, node=node, fail=False)
  File "/buildbot/qa-kvm/qa-kvm-quick/build/qa/qa_utils.py", line 192, in AssertCommand
    _AssertRetCode(rcode, fail, cmdstr, nodename)
  File "/buildbot/qa-kvm/qa-kvm-quick/build/qa/qa_utils.py", line 162, in _AssertRetCode
    (cmdstr, nodename, rcode))
qa_error.Error: Command 'lvs -o tags --noheadings  | grep INSTANCE_NAME' on node NODE_NAME failed, exit code 1

(all-caps text should be substituted with the actual names)

Either the appropriate check is run for non-LVM instances --if applicable--, or tags should be checked only for LVM instances.
```
Originally added on 2013-04-06 09:46:30 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-06 18:50:55 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 1012ecf4e487a0e1525a4c0bef939fe74f9c63e2
Author: Guido Trotter <ultrotter@google.com>
Date:   Sat Apr 6 18:35:34 2013 +0200

    qa: fix rename instance for non-lvm backed storage
    
    The rename instance test performs an lvm-specific check for tags on the
    instance volume. This should be in the future abstracted *inside* the
    template class, and not explicitly called from qa.
    
    This would allow other disks to perform the checks that make sense for
    them. In the meantime though, we just make the same check only when the
    backend storage for the instance is lvm-based.
    
    Signed-off-by: Guido Trotter <ultrotter@google.com>
    Reviewed-by: Bernardo Dal Seno <bdalseno@google.com>

```
Originally added on 2013-04-10 18:13:20 +0000 UTC.

Added Labels: Type-Defect Priority-High
Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
